src/components/Promise/index.js

```diff
@@ -29,6 +29,12 @@ function Promise({
 }) {
   const classes = useStyles({ image: promise.image, classes: classesProp });
 
+  const formatLocation = (latlng) => {
+    if (!latlng) return {};
+    const formatted = latlng.split(",");
+    return { lat: parseFloat(formatted[0]), long: parseFloat(formatted[1]) };
```

Comment:
If the only place this is used is in the radar component, then just leave it as `[lat, long]` array.